### ISSUE TITLE
Duplicate contacts

### ISSUE THREAD

**Issue by FliegendeWurst** [HenriDellal/emerald-dialer#8]
Screenshot of Huawei contacts and Emerald Dialer:

![](https://i.imgur.com/yOxUU52.png) ![](https://i.imgur.com/hakIuME.png)

I suspect that it happens because of the Whatsapp and/or Telegram part.

**Comment by HenriDellal** [comment]
Yes, I have it too. I'd like to add filters, just like every contacts app has.

**Comment by HenriDellal** [comment]
I have added filters with the latest commit, but they aren't automatic and have to be set up in the settings, so I'm not sure if the issue should be closed.
